v21.2/local-testing.md

```diff
@@ -8,16 +8,37 @@ This page documents best practices for unit testing applications built on Cockro
 
 If you are deploying a self-hosted cluster, see the [Production Checklist](recommended-production-settings.html) for information about preparing your cluster for production.
 
+{{site.data.alerts.callout_danger}}
+The settings described on this page are **not recommended** for use in production clusters. They are only recommended for use during unit testing and continuous integration testing (CI).
+{{site.data.alerts.end}}
+
 ## Use a local, single-node cluster with in-memory storage
 
-The [`cockroach start-single-node`](cockroach-start-single-node.html) command starts a single-node, insecure cluster with [in-memory storage](cockroach-start-single-node.html#store):
+The [`cockroach start-single-node`](cockroach-start-single-node.html) command below starts a single-node, insecure cluster with [in-memory storage](cockroach-start-single-node.html#store). Using in-memory storage improves the speed of the cluster for local testing purposes.
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
 cockroach start-single-node --insecure --store=type=mem,size=0.25 --advertise-addr=localhost
 ~~~
 
-Using in-memory storage improves the speed of the cluster for local testing purposes.
+We recommend the following additional [cluster settings](cluster-settings.html) and [SQL statements](sql-statements.html#data-definition-statements) for improved performance during functional unit testing and continuous integration testing:
```

Comment:
Can we add a disclaimer along the lines of `These recommendations do not apply to performance testing since they do not mirror CockroachDB’s actual performance.`